ajax/libs/treesaver/package.json

```diff
@@ -10,5 +10,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Treesaver/treesaver"
-  }
+  },
+  "licenses": [
+    "MIT",
+    "GPL-1.0"
```

Comment:
@lcd78706 
I think it's version 2 https://github.com/Treesaver/treesaver#license